### ISSUE TITLE
Watch later playlist

### ISSUE THREAD

**Issue by GET100PERCENT** [TeamPiped/Piped#1737]
### Describe the feature

It would be better if there is a watch later playlist by default.
And videos on every section can be added to it just by long tapping it and chossing add to watch later.
Also the videos of "watch later" playlist should be exported by the backup option.

### Why would this be useful to add?

When I quickly scroll the feed and see some important videos i have to either download it or play for some seconds in order to add it in the watch history.
So if I can add the video to watch later in just one or two steps it will be very helpful.

### Acknowledgements

- [X] I have searched the existing issues and this is **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this form.

**Comment by IkelAtomig** [comment]
I think, you can do this by creating a Playlist name `Watch Later`. Just tap `+` and select `Watch Later` and add to playlist.

**Comment by GET100PERCENT** [comment]
Yes, as offline playlist feature has been implemented on libretube, now this default watch Later playlist is not much essential.

**Comment by shaedrich** [comment]
Sometimes, when I still used YouTube, I wished, I had multiple "Watch later" playlists. This could
* be an argument against a dedicated "Watch later" playlist
* also be one in favor of it but applying it more broadly to something called "system playlists"/"default playlist"/"special playlist" and might be even used for other (automated) stuff in the future &rarr; implementation would be a simple flag and maybe a setting which flag the user wants to activate
